src/controllers/comments/addCommentsPost.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+const { addCommentsQuery } = require('../../database/queries');
+const { commentSchema } = require('../../utils/validation');
+
+const addCommentsPost = (req, res, next) => {
+  const { comment, postId } = req.body;
```

Comment:
If you can access the `postId` using `params` from the URL will be better than sending it in the request body.